go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	github.com/xlab/treeprint v1.2.0
 	go.uber.org/atomic v1.11.0
 	golang.org/x/mod v0.14.0
-	golang.org/x/net v0.20.0
+	golang.org/x/net v0.23.0
```

Comment:
Looking at the docs the latest version is `v0.24.0`. Should we use that?

Reply:
Sure.